src/Storage/Stores/Serialization/ClaimLite.cs

```diff
@@ -11,4 +11,5 @@ public class ClaimLite
     public string Type { get; set; }
     public string Value { get; set; }
     public string ValueType { get; set; }
-}
\ No newline at end of file
+    public string Issuer { get; set; }
```

Comment:
Any concerns about about existing valves being null ?

Reply:
Nullable reference types aren't enabled in this file, but Issuer and ValueType both in practice can be null and we handle them similarly when we deserialize.